### DISCUSSION TITLE
Filter by Author visual bug on iPad

### DISCUSSION THREAD

**Discussion by tobyab** [community/community#51699] [Mobile]
### Select Topic Area

Bug

### Body

The “Filter by Author” modal has a visual bug on iPad. See attached screenshot

![Filter by Author visual bug](https://user-images.githubusercontent.com/77097223/229307254-0e7580a5-c17d-4e3c-8465-6f726dfea642.jpeg)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.106.0 (94961686)
Device: iPad Pro 11-inch 3 (iOS 16.4)
</details>

Let me know if you need any further information! :)

**Comment by iskae** [comment]
Hello @developedbytoby 👋 

Thanks a bunch for reaching out ❤️ 

Can you elaborate a bit more on the visual bug? 😅 We have struggled a bit to find out the visual bug on this screen 😅 

If the visual bug is the position of the `Done` button, it is expected and in compliance with the guidelines 😊 

Thanks in advance!

**Reply by tobyab** [reply]
Sorry! I was referring to the text saying "Filter by Author".

**Reply by iskae** [reply]
I am so sorry for the back and forth, but we are still struggling to see the visual bug 😅 Is the font different in comparison to other filtering feature types?

**Reply by tobyab** [reply]
It's ok! It may not be a visual bug, however the text, "Filter by Author" looks out of place, since it isn't centered between the buttons; Cancel and Done - instead, it's centered in the modal

**Reply by iskae** [reply]
Ooooh, now it is clear ❤️ 

Unfortunately this is intended/expected since there is not custom logic behind these modals and the modal title is always centered within the modal 😅 

Sorry for not being able to help further on this one and thank you so much for the back and forth🙇

**Reply by tobyab** [reply]
Ah, ok! That's fine. Thank you for the help!! :)
